clappr/clappr-ios
# Remove reload on Container

## Goal

- Remove reload on Container
- Remove play / pause on Container
- Clear `timeObserver` instance on `AVFoundationPlayback` when `removeObservers` is called. (Possible retain cycle in some destroy() scenarios)

## How to Test

- Run all test suite
- On sample app, click in the `Recreate Player` button a lot of times, to ensure that player.destroy() is working well and without any retain cycle.
